schema/vrs/vrs-source.yaml

```diff
@@ -9,7 +9,7 @@
 
 # https://json-schema.org/understanding-json-schema/reference/schema.html
 $schema: "https://json-schema.org/draft/2020-12/schema"
-$id: "https://w3id.org/ga4gh/schema/vrs/2.0.0-snapshot.2025-02.4/vrs-source.yaml"
+$id: "https://w3id.org/ga4gh/schema/vrs/2.0/vrs-source.yaml"
```

Comment:
```suggestion
$id: "https://w3id.org/ga4gh/schema/vrs/2.0.0/vrs-source.yaml"
```

Reply:
see following commit that resolves this for both gks-core and vrs. We should be precise as you suggest.